content/blogs/YYYY-MM-DD-CI-CD-kestra-comparison.md

```diff
@@ -0,0 +1,186 @@
+
+So, while Azure DevOps is great for Microsoft-centric pipelines, Kestra offers the flexibility to handle distributed, cross-cloud workflows with ease.
+
+
+### Kestra vs. Jenkins
+Ah, [Jenkins](https://www.jenkins.io/) — the old reliable. It has been around forever, and it has a [massive ecosystem of plugins](https://plugins.jenkins.io/) to automate almost anything. You can pretty much build anything with Jenkins, but here’s the rub: you’ll need to do a lot of that building yourself, as it requires a lot of configuration; and if you’re managing complex workflows, you’ll likely end up with custom scripts and a mess of plugins.
+
+![A use of Jenkins by Federico Trotta](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/jenkins.png)
+
+So, Jenkins can automate simple tasks like nobody’s business, but Kestra is all about scaling those workflows across systems with less hassle. With Kestra,  in fact, you’re not diving into custom scripting to handle error states or retries—it’s built-in.
+
+So, if your workload involves tasks that involve cloud services, need precise error handling, or have complex dependencies, Kestra handles all that for you out of the box without the plugin bloat or script maintenance headaches as Jenkins would.
+
+
+## Key Features that Make Kestra Stand Out
+By now, you’re surely seeing the pattern: the tools we mentioned are great for CI/CD and managing simple workflows, while Kestra is purpose-built for orchestrating complex workflows across different systems. So, basically, if you're working on the GitHub environment and need a CI solution, you'll use GitHub Actions, and the same thing applies if you're in the GitLab ecosystem.
+
+But what if you need to manage workflows across multiple environments? Here's where Kestra comes into the game!
+
+So, let’s zero in on a few of Kestra’s key features that make it stand out:
+
+- **Tasks Orchestration**: Kestra’s primary strength is in orchestrating tasks across multiple environments. It doesn’t matter if those tasks are happening on different cloud providers, on-prem servers, or in a hybrid environment: Kestra manages the flow of tasks smoothly. For example: CI/CD tools can kick off a build or deploy code, but Kestra can handle workflows that include those steps plus data processing tasks, external API calls, or anything else your workflow requires.
+
+![Workflow orchestration in Kestra](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/kestra_orchestration.jpg)
+
+- **Error Handling and Retries**: In traditional automation tools, error handling and retries often require custom scripting. Kestra, instead, bakes this into its core. In fact, if a task fails, Kestra can automatically retry it, adjust workflows dynamically, or trigger error-specific branches in the workflow. This is a huge time-saver when working with workflows prone to errors, like distributed systems or long-running data processing jobs.
+
+![Managing errors in Kestra](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/kestra_errors.jpg)

```

Comment:
Can you use a different image over a thumbnail. Confusing as these look like the video. If you mean to put the video, then put that. I can make a similar image to the thumbnail if that's what you're after